apps/super-app/README.md

```diff
@@ -39,6 +39,12 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Deployment
+
+* Use `make build` to create the production build.
+* After creating the build, you can load the appropriate environment variables set `source .env.[environment]` from the root folder.
```

Comment:
Is this still accurate? It seems like this is obsolete with the `sh -c` Makefile trick